src/herder/test/HerderTests.cpp

```diff
@@ -1890,7 +1890,7 @@ TEST_CASE("quick restart", "[herder][quickRestart]")
     auto static const SMALL_GAP = app->getConfig().MAX_SLOTS_TO_REMEMBER - 1;
     // BIG_GAP, we just need to pick a number greater than what we keep in
     // memory
-    auto static const BIG_GAP = app->getConfig().MAX_SLOTS_TO_REMEMBER + 1;
+    auto static const BIG_GAP = app->getConfig().MAX_SLOTS_TO_REMEMBER + 2;
```

Comment:
@marta-lokhova I had to increase this for the test to pass.

Reply:
weird, I don't see why this would be the case. there are no out of order ledgers. @marta-lokhova any idea?

Reply:
hah didn't realize the test is failing; this is related to cfg1, cfg2 fix on line 1841-1842. turns out there is an [off-by-one error](https://github.com/stellar/stellar-core/blob/master/src/herder/HerderImpl.cpp#L234) in ledger validity bracket implementation: we remember `MAX_SLOTS_TO_REMEMBER + 1`  This test was not failing before because listener was 1 ledger behind already before validator started "creating the gap", so it'd fail. So we should fix the math in Herder, and leave "big gap" as is.